internal/driver/fetch.go

```diff
@@ -41,22 +41,29 @@ import (
 // there are some failures. It will return an error if it is unable to
 // fetch any profiles.
 func fetchProfiles(s *source, o *plugin.Options) (*profile.Profile, error) {
+	normalize := s.Normalize
+	if normalize && len(s.Base) == 0 {
+		return nil, fmt.Errorf("base profile required when normalizing")
+	}
+
 	sources := make([]profileSource, 0, len(s.Sources)+len(s.Base))
 	for _, src := range s.Sources {
 		sources = append(sources, profileSource{
 			addr:   src,
 			source: s,
 			scale:  1,
+			isBase: false,
 		})
 	}
 	for _, src := range s.Base {
 		sources = append(sources, profileSource{
 			addr:   src,
 			source: s,
 			scale:  -1,
+			isBase: true,
 		})
 	}
-	p, msrcs, save, cnt, err := chunkedGrab(sources, o.Fetch, o.Obj, o.UI)
+	p, msrcs, save, cnt, err := chunkedGrab(sources, normalize, o.Fetch, o.Obj, o.UI)
```

Comment:
Instead of having to pass isBase in, could we invoke chunkedGrab separately for the input profiles and the base profiles (probably in parallel), then do the normalization of the merged input over the merged base, then combine the two profiles returning the result.